cpp/log/sqlite_db-inl.h

```diff
@@ -21,7 +23,14 @@ DEFINE_bool(sqlite_synchronous_off, false,
 namespace {
 
 
+static cert_trans::Latency<std::chrono::milliseconds, std::string>
+    latency_by_op_ms("database_latency_by_operation_ms", "operation",
```

Comment:
By which I mean, there *are*, and it's a bit confusing having two variables with the same name (I'm not even 100% on whether that's allowed, or if it should even be).